Jenkinsfile

```diff
@@ -14,15 +14,15 @@ goTemplate{
           dockerOrganisation = 'fabric8'
           project = 'fabric8-auth'
           dockerBuildOptions = '--file Dockerfile.deploy'
-          makeTarget = 'build check-go-format test-unit-junit'
```

Comment:
Fair enough, we don't need fabric8 ci infra to test. Only the images are helpful.